tests/phpunit/tests/functions/TestConfigWpHome.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Test the _config_wp_home function
+ *
+ * @group Functions
+ * @covers ::_config_wp_home
+ */
+class Test_for_config_wp_home extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 57179
+	 * @dataProvider _config_wp_home_dataset
+	 */
+	function test_config_wp_home( $url, $expected ) {
+		$this->assertSame( $expected, _config_wp_home( $url ) );
+	}
+
+	function _config_wp_home_dataset() {
```

Comment:
```suggestion
	/**
	 * Data provider for test_config_wp_home().
	 *
	 * @return array[]
	 */
	function data_config_wp_home() {
```
- Data providers should also have a docblock.
- Data providers should have the `data_` prefix. See [this ticket](https://core.trac.wordpress.org/ticket/53119).